code/game/gamemodes/objective_items.dm

```diff
@@ -25,50 +27,127 @@
 	GLOB.possible_items -= src
 	return ..()
 
+// Low risk steal objectives
+/datum/objective_item/steal/low_risk
+	objective_type = OBJECTIVE_ITEM_TYPE_TRAITOR
+
+/datum/objective_item/steal/low_risk/techboard
+	name = "the (TECH BOARD) circuitboard in secure tech storage"
+	var/circuitboard_name
+	excludefromjob = list(
+		"Captain",
+		"Chief Engineer",
+		"Research Director",
+	)
+
+/datum/objective_item/steal/low_risk/techboard/aiupload
+	targetitem = /obj/item/circuitboard/computer/aiupload
+	circuitboard_name = "ai upload"
+
+/datum/objective_item/steal/low_risk/techboard/borgupload
+	targetitem = /obj/item/circuitboard/computer/borgupload
+	circuitboard_name = "cyborg upload"
+
+/datum/objective_item/steal/low_risk/techboard/New()
+	. = ..()
+	name = replacetext(name, "(TECH BOARD)", circuitboard_name)
+
+/datum/objective_item/steal/low_risk/aicard
+	targetitem = /obj/item/aicard
+	name = "an aicard"
```

Comment:
This isn't what it's called in game, is it?

Frankly we should be automatically filling the name in with `initial(name)` by default if we're not already.

Reply:
there's custom names for each steal item, like the nuke core and stuff